gdxsv/lbs_discord.go

```diff
@@ -0,0 +1,494 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"net/http"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+	"sync/atomic"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+var retryTimer *time.Timer
+var jobIsRunning uint32
```

Comment:
These variables are exposed by the main package. Let's use a more specific name.

Reply:
Renamed to
```go
var discordLiveStatusRetryTimer *time.Timer
var discordLiveStatusIsPublishing uint32
```

All other type definitions and helper functions are not exposed now